kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -51,9 +51,13 @@ def build(self):
                                  "have exactly one element")
         else:
             raise ValueError("shape must be an integer or a 1-tuple")
-        
+
         if self.T is None:
             self.T = 2**(self.J)
+        elif self.T > 2**(self.J):
+            raise ValueError("The temporal support T of the low-pass filter "
```

Comment:
Why?

Reply:
[See 2](https://github.com/kymatio/kymatio/pull/673#pullrequestreview-626744057). Should there be _no_ constraint on `T`? Unsure how this plays out with `T >> N`, but more work needs to be done.

Reply:
Constraint should be whatever was in place previously for `2 ** J`.

Reply:
Then that's what this accomplishes. The idea's with this PR is to allow `T != 2**J`, but implementation has catching up to do if `T > 2**J`.

Reply:
But having `T > 2 ** J` is desirable in certain cases (@lostanlen does this quite often, from what I recall). The constraint before was that `2 ** J` cannot be much larger than `N` (I forget what the exact limit was), that same constraint should be applied to `T` instead.

Reply:
Looks like this is resolved?

Reply:
> Old version is highlighted, see [latest](https://github.com/kymatio/kymatio/blob/1ee37e84ac3c0050440f64910a395fab97a30bf8/kymatio/scattering1d/frontend/base_frontend.py#L58).

Ok. Is that a yes or no?

Reply:
Yes resolved